Automattic/vip-go-mu-plugins
# Update ElasticPress to latest

## Description

Update the ElasticPress submodule to the latest from `Automattic/ElasticPress#master`

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Check out PR.
1. Index content - `wp vip-search index`
1. Search content = `/?s=test`
1. Profit $$$
